instructor/cli/files.py

```diff
@@ -1,6 +1,8 @@
+# type: ignore - stub mismatched
```

Comment:
We're really not following the stubs at all here, so not too sure if this was just from when we were on a previous version of the OpenAI client? Mainly, that we treat the pydantic models they return as dictionaries, but maybe there's something I'm missing. Thought it would be easiest to ignore this for now, and go back to it later.